backend/src/main/java/atlantbh/restaurants/repositories/MenuRepository.java

```diff
@@ -0,0 +1,20 @@
+package atlantbh.restaurants.repositories;
+
+import atlantbh.restaurants.models.Menu;
+import atlantbh.restaurants.models.filters.MenuFilterBuilder;
+import atlantbh.restaurants.models.sortkeys.MenuSortKeys;
+import org.hibernate.Criteria;
+import org.hibernate.criterion.Restrictions;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public class MenuRepository extends BaseRepositoryImpl<Menu, MenuSortKeys, MenuFilterBuilder> {
+    public List<Menu> findByRestaurant(Long restaurant) {
+        Criteria criteria = getBaseCriteria();
+        criteria.createAlias("restaurant", "restaurant");
+        criteria.add(Restrictions.eq("restaurant.id", restaurant));
```

Comment:
remove alias

Reply:
Removed obsolete method in 0cd4debab433260ab6a8267aa0b72b31db581edb